pegasus/sites.v3/code.org/public/maker/circuitplayground.haml

```diff
@@ -0,0 +1,97 @@
+---
+title: The Circuit Playground | Code.org
+theme: responsive_full_width
+---
+
+%link{href: '../shared/css/design-system-pegasus.css', rel: 'stylesheet'}
+%link{href: '/css/generated/maker-styles.css', rel: 'stylesheet'}
+
+-# Import FontAwesome Pro
+%script{src: "https://kit.fontawesome.com/ea2d9d4413.js", crossorigin: "anonymous"}
```

Comment:
Wow, do you know if we can do this on Dashboard?

Reply:
If only it were so simple haha! This is sort of a long story: this implementation is pulling from an eternal url which is ok for Pegasus since the icons are all decorative so if it gets blocked and the icons disappear it won't affect the UX at all — but since some schools block external urls it's better to have Font Awesome come from our codebase instead on Dashboard since the icons there are more important/informational. 

Ideally, once Font Awesome gets implemented internally we'll switch over to use that here for Pegasus too, but in the meantime this is an ok workaround.

Reply:
Oh, that's good to know! Your explanation makes a lot of sense –– In a future PR, I might go ahead and add it to get the design style right and add a todo to change it back later.